src/Symfony/Installer/NewCommand.php

```diff
@@ -356,6 +358,22 @@ private function cleanUp()
     {
         $this->fs->remove(dirname($this->compressedFilePath));
 
+        try {
+            $licenseFile = array($this->projectDir.'/LICENSE');
+            $upgradeFiles = glob($this->projectDir.'/UPGRADE*.md');
+            $changelogFiles = glob($this->projectDir.'/CHANGELOG*.md');
```

Comment:
@javiereguiluz What do you think about `*.md` then? You will probably not want to keep any markdown files from the SE at all.